src/components/BulkActionsToolbar/BulkActionsToolbar.tsx

```diff
@@ -73,7 +74,7 @@ export const BulkActionsToolbar: React.FC<Props> = (props) => {
                     // be able to show the tooltip on disabled button.
                     // https://github.com/atomiks/tippyjs-react/issues/123
                     return (
-                        <Tooltip key={action.label} content={action.tooltip}>
+                        <Tooltip key={action.label} content={action.tooltip} delay={action.delay}>
```

Comment:
`delay` is optional and I'm wondering how `Tooltip` component will behave if the value will be `undefined`.

Reply:
I am not sure, but visually it works fine